priv/repo/seeds.exs

```diff
@@ -40,27 +41,31 @@ defmodule Seed do
         city: Address.city(),
         state: Address.state(),
         pincode: Address.postcode(),
-        role: "admin"
+        role: "admin",
+        whatsapp_phone: Phone.PtPt.number(),
+        date_of_birth: Faker.Date.date_of_birth(Enum.random(1..10))
       })
 
     user
   end
 
-  def create_batch() do
-    {:ok, batch} =
-      Batches.create_batch(%{
-        name: Address.city() <> " " <> "Batch"
-      })
-
-    batch
-  end
-
   def create_group() do
     {:ok, group} =
       Groups.create_group(%{
-        input_schema: %{},
-        locale: Enum.random(["hi", "en"]),
-        locale_data: %{}
+        name: Person.name(),
+        type: Enum.random(["batch", "group", "cohort", "program"]),
+        program_type: Enum.random(["Competitive", "Non Competitive"]),
+        program_sub_type: Enum.random(["Easy", "Moderate", "High"]),
+        program_mode: Enum.random(["Online", "Offline"]),
+        program_start_date: Faker.DateTime.backward(Enum.random(1..10)),
+        program_target_outreach: Enum.random(["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]),
```

Comment:
let's make this a values in thousands. Like how many thousand students the program is targeting. 1 to 10 is way less